docs/docs-content/vm-management/vm-migration-assistant/migrate-vms-vmo-cluster.md

```diff
@@ -0,0 +1,369 @@
+---
+sidebar_label: "Migrate VMs to a VMO cluster"
+title: "Migrate VMs to a VMO cluster using the VM Migration Assistant"
+description: "Learn how to migrate VMs to Palette VMO using the VM Migration Assistant"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo", "vm migration assistant"]
+#toc_max_heading_level: 4
```

Comment:
We can adjust the layout so that Migrate VMs is an H2, and Cancel Active Migration

![CleanShot 2024-11-22 at 13 24 08](https://github.com/user-attachments/assets/b1d737b1-55e6-4160-8a8d-094f81173fcf)